### ISSUE TITLE
[FR] Command to remove database engine

### ISSUE THREAD

**Issue by mattstein** [craftcms/nitro#325]
### Description

There should be a command to remove a database container, because I trust Nitro to help me remove anything it helps me add.

Right now I can hand-edit `~/.nitro/nitro.yaml` to remove the relevant `databases` entry and run `apply`, and it works despite the slightly uncomfortable output:

```sh
# (...) unrelated stuff happens
Cleaning up...
  … removing mariadb-latest-3307.database.nitro   … creating backup @Database-2021-04-05-093134.sql ✗
Unable to backup database @Database Error response from daemon: lstat /var/lib/docker/overlay2/e15fe6056339dadf4cad6bd7815d0da3f4f435f3f00499faad3cd85120c0d5ae/merged/tmp/@Database-2021-04-05-093134.sql: invalid argument
Backups saved in /Users/mattstein/.nitro/mariadb-latest-3307.database.nitro 💾
# (...) unrelated cleanup finishes
```

Output could have been my fault, but I’d _know_ who bug from PEBKAC if I was using `nitro db destroy` or `nitro db remove-engine`.

**Comment by jasonmccallister** [comment]
@mattstein I think `nitro db destroy` makes more sense as it is something that is destructive and potentially could result in data loss.

**Comment by jasonmccallister** [comment]
Released with v2.0.8.
